Resources/Locale/en-US/_NF/mail/mail.ftl

```diff
@@ -0,0 +1,9 @@
+mail-large-item-name-unaddressed = package
+mail-large-item-name-addressed = package ({$recipient})
+mail-large-desc-far = A large package.
+mail-large-desc-close = A large package addressed to {CAPITALIZE($name)}, {$job}. Last known location: {$station}.
+
+# Frontier: mailtestbulk
```

Comment:
```suggestion
```

.ftl doesn't support comments and this causes one of the 2000+ warnings to pop up when the game builds

Reply:
Oh my lord. Okay!

Reply:
> .ftl doesn't support comments and this causes one of the 2000+ warnings to pop up when the game builds

It does, it's just that people can't format them properly. https://projectfluent.org/fluent/guide/comments.html